app/Http/Controllers/UsersController.php

```diff
@@ -250,20 +246,34 @@ public function show($id, $mode = null)
                 'Achievement'
             );
 
+            $perPage = [
+                'scoresBest' => 5,
+                'scoresFirsts' => 5,
+                'beatmapPlaycounts' => 5,
+                'scoresRecent' => 5,
+
+                'favouriteBeatmapsets' => 6,
+                'rankedAndApprovedBeatmapsets' => 6,
+                'unrankedBeatmapsets' => 6,
+                'graveyardBeatmapsets' => 2,
+
+                'recentlyReceivedKudosu' => 5,
+            ];
+
             $beatmapsets = [
-                'most_played' => $this->mostPlayedBeatmapsets($user),
-                'ranked_and_approved' => $this->rankedAndApprovedBeatmapsets($user),
-                'favourite' => $this->favouriteBeatmapsets($user),
-                'unranked' => $this->unrankedBeatmapsets($user),
-                'graveyard' => $this->graveyardBeatmapsets($user),
+                'most_played' => $this->mostPlayedBeatmapsets($user, $perPage['beatmapPlaycounts'] + 1),
```

Comment:
probably worth adding a comment on why the `+ 1` is necessary, for future explorers of this code.